examples/freestyle/src/frontend/App.tsx

```diff
@@ -0,0 +1,100 @@
+import { createClient, createRivetKit } from "@rivetkit/react";
+import { useEffect, useState } from "react";
+import type { Message, registry } from "../backend/registry";
+
+const client = createClient<typeof registry>("http://localhost:8080/");
```

Comment:
needs to read VITE_RIVET_ENGINE and VITE_RIVET_TOKEN and callback to undefined. when you run freestyle deploy, you'll need to autopouplate this env. might require moving tsup in to the deploy script.